sim/battle-stream-example.ts

```diff
@@ -99,15 +91,18 @@ const p2spec = {
 const p1 = new RandomPlayerAI(streams.p1);
 const p2 = new RandomPlayerAI(streams.p2);
 
+/* tslint:disable:no-console */
 console.log("p1 is " + p1.constructor.name);
 console.log("p2 is " + p2.constructor.name);
 
 (async () => {
 	let chunk;
+	// tslint:disable-next-line no-conditional-assignment
```

Comment:
Doesn't tslint have the feature to allow conditional assignment when wrapped in parens?

Reply:
Apparently not :(. Or maybe I'm missing it?